### DISCUSSION TITLE
Specific reason for using // comments instead of /**/ ?

### DISCUSSION THREAD

**Discussion by SylvainSoKette** [libuv/libuv#3666] [Q&A] [answered]
I'm currently failing to build [mediasoup](https://github.com/versatica/mediasoup) which relies on libuv. The errors comes up when gcc encounters some C++ style comments in ```src/unix/stream.c```

I grepped around and found some other occurences in ```src/win/tcp.c``` and ```src/unix/tty.c```, but those are either plateform specific or behind a ```#define``` and I probably won't encounter them myself.

Since the CMakeLists.txt defines the project as using the C90 standard (```set(CMAKE_C_STANDARD 90)```), is this a mistake that should be fixed or is there a specific reason for using this kind of comment in just a few places ?

**Comment by bnoordhuis** [comment, marked as answer]
They're a mistake. I've opened #3667 to fix them.
